yarn.lock

```diff
@@ -2349,7 +2349,7 @@
     "@docsearch/css" "3.1.1"
     algoliasearch "^4.0.0"
 
-"@docusaurus/core@2.2.0", "@docusaurus/core@^2.2.0":
+"@docusaurus/core@2.2.0":
```

Comment:
I would have expected 2.2.0 and its dependencies to get dropped. Is that ok?

Reply:
Good question. I think yarn should does that. @jordojordo do you know if this is okay?

Reply:
It's possible `"@docusaurus/theme-search-algolia": "^2.2.0"` not being bumped is causing this.

Reply:
@btat is right, running `yarn list @docusaurus/core` yields:
```console
$ yarn list @docusaurus/core
yarn list v1.22.19
├─ @docusaurus/core@2.3.1
└─ @docusaurus/theme-search-algolia@2.2.0
   └─ @docusaurus/core@2.2.0
```

Reply:
I fixed that